docs/integration/DataTypingAndSchemasHandling.md

```diff
@@ -0,0 +1,246 @@
+| Integer                                                      | date (int)                               |                                                                                                                   |
+| LocalTime                                                    | time - millisecond precision (int)       |                                                                                                                   |
+| Integer                                                      | time - millisecond precision (int)       |                                                                                                                   |
+| LocalTime                                                    | time - microsecond precision (long)      |                                                                                                                   |
+| Long                                                         | time - microsecond precision (long)      |                                                                                                                   |
+| Instant                                                      | timestamp - millisecond precision (long) |                                                                                                                   |
+| Long                                                         | timestamp - millisecond precision (long) |                                                                                                                   |
+| Instant                                                      | timestamp - microsecond precision (long) |                                                                                                                   |
+| Long                                                         | timestamp - microsecond precision (long) |                                                                                                                   |
+| Any matching type from the list of types in the union schema | union                                    | Read more about [validation modes](#validation-and-encoding).                                                     |
+
+If at runtime value cannot be converted to an appropriate logic schema (e.g. "notAUUID" cannot be converted to
+proper UUID), then an error will be reported.
+
+## JSON Schema
+
+We support [JSON Schema](https://json-schema.org/) in version: `Draft 7` without:
+
+[//]: # (See SwaggerBasedJsonSchemaTypeDefinitionExtractor)
+
+* Numbers with a zero fractional part (e.g. `1.0`) as a proper value on decoding (deserialization)
+  for [integer schema](https://json-schema.org/understanding-json-schema/reference/numeric.html#integer)
+* Nested relative schemas
```

Comment:
What is 'nested relative schema'?

Reply:
Okej, I've modified this list a little bit :)